treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -20,7 +20,7 @@
     keywords: perf-alert, regression
     default_component: Performance
     default_product: Testing
-    cc_list: nfroyd@mozilla.com
+    cc_list: froydnj+bz@gmail.com
```

Comment:
I don't think Nathan is working at Mozilla anymore. We should leave the `cc_list` blank instead.